Week-6/windowsObject/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Window Object Properties</title>
+    <script src="script.js" defer></script>
+</head>
+<body onload="displayWindowProperties()">
```

Comment:
Since `window.onload` is used at the bottom of the script, the `onload `attribute in the `body `tag is redundant and could be removed.